static/locales/ja.json

```
@@ -1577,8 +1579,8 @@
             "vllm_omni": "vLLM-Omni（ローカルクローン）"
         },
         "sourcePreset": "プリセット",
-        "sourceClone": "クローン",
-        "sourceDesign": "説明から生成",
+        "sourceClone": "音声クローン",
```

Biome (2.5.1)

[error] The key sourceClone was already declared.

(lint/suspicious/noDuplicateObjectKeys)